### ISSUE TITLE
transformer implementation error

### ISSUE THREAD

**Issue by chengmengli06** [tensorflow/tensorflow#32167]
https://github.com/tensorflow/models/blob/master/official/transformer/model/transformer.py
The transfomer implementation seems to be incorrect. I cannot find the residual component, and the batchnorm is missed after every feed forward layer.

**Comment by ymodak** [comment]
You may post this issue on tensorflow models repository since it belongs to that repository. Thanks!
https://github.com/tensorflow/models/issues
